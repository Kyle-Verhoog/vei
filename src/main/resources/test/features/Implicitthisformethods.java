public class Implicitthisformethods {
  public Implicitthisformethods() {}
  public int m1() {
    return 42;
  }
  public int m2() {
    return m1();
  }
}