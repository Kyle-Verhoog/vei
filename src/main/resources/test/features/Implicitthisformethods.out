PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
INT ┃ int
IDENTIFIER ┃ m1
( ┃ (
) ┃ )
{ ┃ {
RETURN ┃ return
INTEGER_LITERAL ┃ 42
; ┃ ;
} ┃ }
PUBLIC ┃ public
INT ┃ int
IDENTIFIER ┃ m2
( ┃ (
) ┃ )
{ ┃ {
RETURN ┃ return
IDENTIFIER ┃ m1
( ┃ (
) ┃ )
; ┃ ;
} ┃ }
} ┃ }
